doc/manual/migrating.rst

```diff
@@ -0,0 +1,53 @@
+.. _migrating
+
+Migrating from EEGLAB
+=====================
+
+To read in data exported from EEGLAB, MNE offers an EDF reader :func:`mne.io.read_raw_edf` and a `set` file reader.
+To read in `set` files containing ``raw`` data, use :func:`mne.io.read_raw_eeglab` and to read in `set` files containing
+``epochs`` data, use :func:`mne.read_epochs_eeglab`.
+
+Here is a cheatsheet to help users migrate painlessly from EEGLAB. For the sake of clarity, let us assume
+that the following are already defined or known: the file name ``fname``, low-pass cut-off frequency ``lfreq``,
```

Comment:
low-pass cut-off frequency ``lfreq`` -> Low cut-off frequency in Hz ``l_freq``

the l_freq sets the cut-off for the high pass filter.